charts/internal/events/templates/deployment.yaml

```diff
@@ -36,6 +41,17 @@ spec:
       containers:
         - name: kube-state-events
           image: {{ .Values.image.kube_state_events.repository }}:{{ default .Chart.AppVersion .Values.image.kube_state_events.tag }}
+          imagePullPolicy: {{ .Values.image.kube_state_events.imagePullPolicy | default "IfNotPresent" }}
+          {{- if .Values.image.kube_state_events.command }}
+          command: {{- range .Values.image.kube_state_events.command }}
```

Comment:
Do you have a use case in mind for overriding the command and args?

Reply:
Yep. Our use case. We use some sort of "wrapper" to inject sensitive information/variables into pods. In order to have this wrapper executed, we need to run a different entrypoint. This change won't affect the current behavior but will allow us (and other possible customers) to make these customizations without having to fork and maintaining their own version of it (like we're doing right now)